src/index.js

```diff
@@ -3,7 +3,7 @@
 import $ from 'jquery';
 import chai from 'chai';
 import testSuiteSkeleton from './assets/test-suite-skeleton';
-import mochaCSS from './assets/mocha-CSS';
+import mochaCSS from './stylesheets/style.css';
```

Comment:
As we discussed via Skype, maybe this variable could have a more accurate name than `mochaCSS`? Something like `testSuiteFCCStyles`?